data/add_coordinates.py

```
@@ -0,0 +1,210 @@
+        response = requests.get(url, params=params, headers=headers, timeout=10)
+        response.raise_for_status()
+
+        data = response.json()
+
+        if data and len(data) > 0:
+            lat = float(data[0]["lat"])
+            lon = float(data[0]["lon"])
+            print(f"  ✅ Знайдено: {lat:.6f}, {lon:.6f}")
+            return (lat, lon)
+        else:
+            print(f"  ❌ Не знайдено в API")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)